src/core/utils/urlutils.js

```diff
@@ -80,3 +84,60 @@ export function equivalentParams (params1, params2) {
   }
   return true;
 }
+
+/**
+ * Creates a copy of the provided {@link SearchParams}, with the specified
+ * attributes filtered out
+ * @param {SearchParams} params The parameters to remove from
+ * @param {string[]} prefixes The prefixes of parameters to remove
+ * @return {SearchParams} A new instance of SearchParams without removed params
```

Comment:
nit but can we reword slightly we say params 3 times in a row here lol

Reply:
😂